index.js

```diff
@@ -1,39 +1,76 @@
-const bookList = document.querySelector('#bookList');
-const addBtn = document.getElementById('addBtn');
-let collection = JSON.parse(localStorage.getItem('bookCollection')) || [];
-
-function renderCollection() {
-  let html = '';
-  collection.forEach((element, index) => {
-    const removeBtn = `<button onclick='removeBook(${index})'>Remove</button>`;
-    html += `<div class='book'>
-        <div>${element.title}</div>
-        <div>${element.author}</div>
-        ${removeBtn}
-        <hr>
-        </div>`;
-  });
-  bookList.innerHTML = html;
-}
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+import currentTime from './modules/showtime.js';
+import { addClick, contactClick, listClick } from './modules/operations.js';
+import variables from './modules/link.js';
 
-// eslint-disable-next-line no-unused-vars
-function removeBook(bookIndex) {
-  collection = collection.filter((element, index) => index !== bookIndex);
-  localStorage.setItem('bookCollection', JSON.stringify(collection));
-  renderCollection();
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
+class ShowBook {
+  constructor() {
+    this.bookList = variables.booklist;
+    this.addBtn = variables.addBtn;
+    this.collection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+    this.addBtn.addEventListener('click', () => {
+      this.addBook();
+    });
+    this.renderCollection();
+  }
+
+  renderCollection() {
+    let html = '';
+    this.collection.forEach((element) => {
+      html += `<div class='book'>
+            <span class=' white lists-header'>
+             <p class='title-header bold'>${element.title}</p>
+             <p class='author-header bold'>${element.author}</p>
+             <button class='RemoveButton'>Remove</button>
+         </span>
+                 </div>`;
+    });
+    this.bookList.innerHTML = html;
+    this.removeBtns = document.querySelectorAll('.RemoveButton');
+    this.removeBtns.forEach((element, index) => {
+      element.addEventListener('click', () => {
+        this.removeBook(element, index);
+      });
+    });
+  }
+
+  addBook() {
+    if (variables.Title.value && variables.Author.value) {
+      this.book = new Book(variables.Title.value, variables.Author.value);
+      this.collection.push(this.book);
+      localStorage.setItem('bookCollection', JSON.stringify(this.collection));
+      this.renderCollection();
+      variables.Title.value = '';
+      variables.Author.value = '';
+    }
```

Comment:
- [ ] Great job working on the functionality so far but I think you should Refactor your functions to arrow functions. so please as required refactor these method to be arrow methods and all the methods in your project to be arrow methods that's because you should learn the ES6 syntax in this milestone✨💯😊🚀